src/main/java/nextstep/courses/domain/Session.java

```diff
@@ -0,0 +1,86 @@
+package nextstep.courses.domain;
+
+import nextstep.courses.domain.field.CoverImage;
+import nextstep.courses.domain.field.Period;
+import nextstep.courses.domain.field.SessionStatus;
+import nextstep.courses.domain.field.SessionType;
+import nextstep.payments.domain.Payment;
+
+import java.time.LocalDateTime;
+import java.util.concurrent.atomic.AtomicLong;
+
+public class Session {
+    private static AtomicLong autoGenId = new AtomicLong(1L);
```

Comment:
```suggestion
    private static AtomicLong AUTO_GEN_ID = new AtomicLong(1L);
```

상수는 대문자패턴으로 사용하는 편입니다.

`Gen`이 무슨의미인지 몰라서 그런데 혹시 축약일까요?
>### 객체지향 생활 체조 원칙
- 규칙 5: 줄여쓰지 않는다(축약 금지).

Reply:
말씀하신대로 대문자패턴과 풀네임으로 변경하였습니다~!